doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+
+#### Creating a subscriber
+
+We now create a corresponding subscriber, usually in another application (it will work in the same application as
+well but there is no need sending it via the middleware in such a case).
+
+```
+iox::popo::TypedSubscriber<CounterTopic> subscriber({"Group", "Instance", "CounterTopic"});
+subscriber.subscribe();
+```
+The template data type and the three string identifiers have to match those of the publisher, in other words the
+service descriptions have to be the same (otherwise we will not receive data from our publisher).
+
+We immediately subscribe here, but this can be postponed to the point were we actually want to receive data.
+
+#### Receiving data
+For simplicity we assume that we periodically check for new data. It is also possible to explicitly wait for data
+using the [Waitset](../../../iceoryx_examples/waitset/README.md). The code to recieve the data is the same, the only
+difference is the way we wake up before checking for data.
+```
+while(keepRunning) {
+    //wait for new data (either sleep and wake up periodically or by notification from the waitset)
+
+    subscriber->take()
+        .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
+            CounterTopic* ptr = sample.get();
+            /* process the received data using the ptr */
+
+            /* alternatively use operator-> */
+            uint32_t counter = sample->counter;           
+        })
+        .if_empty([] { /* no data received but also no error */ })
+        .or_else([](iox::popo::ChunkReceiveError) { /* handle the error */ });
+}
+```
+
+By calling ``take`` we get a ``iox::cxx::expected<iox::optional<iox::popo::Sample<const CounterTopic>>>``. Since this
+may fail, we handle a potential error in the ``or_else`` branch. If we wake up periodically, it is also possible that
+no data is received and if we want to handle this we can optionally do so in the ``if_empty`` branch.
+
+The usual case is that we actually receive data, and we process it in the ``and_then``. Notice that in the lambda we
+do not pass a ``CounterTopic`` directly but a reference to a ``iox::popo::Sample<const CounterTopic>&``. We can
+access the underlying ``CounterTopic`` either by getting a pointer to it via ``get`` or by using ``operator->``. In
+any case, we now can process or copy the received data and once the ``sample`` goes out of scope, the underlying
+``CounterTopic`` object is deleted as well (this happens when the temporary object returned by ``take`` is
+destroyed). This means it is only safe to hold references to the data as long as the ``sample`` exists. Should we
+need a longer lifetime, we have to copy or move the data from the ``sample``.
+
+This only allows us to get one sample at a time. Should we want to get all currently available samples we can do so
+by using an additional loop.
+
+```
+while(keepRunning) {
+    while(subscriber.hasNewSamples()) {
```

Comment:
to to self: Check if we have an example for that

Reply:
I use it in the multi publisher example, so yes.